domains/mousemoosz.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "MouseMoosz",
+    "email": "mousemoosz@protonmail.com"
+    "description": "For my website"
+  }
+  "record": {
+    "CNAME": "mousemoosz.netlify.app"
```

Comment:
sorry netlify doesnt work with this service =(

Reply:
Oh, well are there no workarounds either?

Reply:
you can try railway or github pages.

all of them are great 👍

Reply:
Alright, I will use github pages.